starter/js/atm.js

```diff
@@ -1,5 +1,89 @@
 //Begin with the document ready function
-
+$(document).ready(function() {
+  $('#depositChecking').on('click',deposOrWithdraw);
+  $('#withdrawChecking').on('click',deposOrWithdraw);
+  $('#depositSavings').on('click',deposOrWithdraw);
+  $('#withdrawSavings').on('click',deposOrWithdraw);
+})
+
+function makeANum(string) {
```

Comment:
NICE! I like that you turned this into a function itself since it's being used more than once!